packages/frontend/src/App.tsx

```diff
@@ -154,6 +155,16 @@ const App = () => (
                                                                 <ProjectSettings />
                                                             </TrackPage>
                                                         </Route>
+                                                        <Route path="/projects/:projectUuid/generalSettings/:tab?">
```

Comment:
General settings aren't specific to a project and can be outside the <AppRoute/> since it can be accessed when the user has no projects access.  

```suggestion
                                                        <Route path="/generalSettings/:tab?">
```